### PR TITLE
Drop the "immutable" term

### PR DESCRIPTION
**Pull request by cgwalters**
I'm still waging a fight against this.  Please stop saying "immutable"
for this, it's more misleading than useful.

Previously e.g. https://github.com/debarshiray/toolbox/pull/43

### CODE REVIEW

**Comment by HarryPhoon** [comment]
Hi @cgwalters, you put an extra space after the end of sentence on line 12.

**Comment by cgwalters** [comment]
> Hi @cgwalters, you put an extra space after the end of sentence on line 12.

OK, fixed.

File: profile.d/toolbox.sh [hunk 1]
```diff
@@ -9,8 +9,8 @@ if [ -f /run/ostree-booted ] \
    && ! [ -f "$host_welcome_stub" ]; then
     echo ""
     echo "Welcome to Fedora Silverblue. This terminal is running on the"
-    echo "immutable host system. You may want to try out the Toolbox for a"
-    echo "more traditional environment that allows package installation"
+    echo "host system. You may want to try out the Toolbox for a"
+    echo "directly mutable environment that allows package installation"
```
Comment (debarshiray) [inline profile.d/toolbox.sh]: Nitpick: would be nice to realign this since this is a user-visible blurb.

Comment (debarshiray) [reply profile.d/toolbox.sh]: I took the liberty to adjust it.

**Pull request merged**